ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/GenesisGenerator.java

```diff
@@ -44,8 +44,8 @@
 import tech.pegasys.artemis.util.hashtree.HashTreeUtil;
 
 public class GenesisGenerator {
-  private final MerkleTree<Bytes32> depositMerkleTree =
-      new MerkleTree<>(DEPOSIT_CONTRACT_TREE_DEPTH);
+  private final OptimizedMerkleTree merkleTree =
```

Comment:
I probably wouldn't rename this - `depositMerkleTree`  includes the important information that this is for deposits, not just that it's a merkle trie (side note: should be `trie` everywhere not `tree` but that can be for another day)

Reply:
Done.